control-plane/connect-inject/controllers/endpointsv2/endpoints_controller_test.go

```diff
@@ -169,17 +169,17 @@ func TestReconcile_CreateService(t *testing.T) {
 							Addresses: addressesForPods(pod1, pod2),
 							Ports: []corev1.EndpointPort{
 								{
-									Name:        "my-http-port",
+									Name:        "public",
```

Comment:
`ServicePort` and `EndpointPort` names are supposed to match in K8s; noticed these were incorrect so fixing along w/ added test.